src/lib/server/events/gcal.ts

```diff
@@ -1,36 +1,30 @@
-import * as RRule from 'rrule/dist/es5/rrule.min.js';
 import { Temporal } from '@js-temporal/polyfill';
 import { TEAMS } from '$lib/public/board/data';
-import { parseBool } from '$lib/server/parse-bool/parse-bool';
 import type { ClubEvent } from '$lib/public/events/event';
+import { google } from 'googleapis';
+import { config } from 'dotenv';
+import { destroy_each } from 'svelte/internal';
 
 export interface ICALParseOptions {
   filterBefore?: boolean;
   referenceDate?: Temporal.ZonedDateTime;
   maxEvents?: number;
 }
 
-export function parse(rawICAL: string, options?: ICALParseOptions): ClubEvent[] {
-  const events: ClubEvent[] = [];
-
+export async function fromGCal(
```

Comment:
Operation `fromGCal` is synchronous.
```suggestion
export function fromGCal(
```